core/modules/css/python/testKvInterface.py

```diff
@@ -27,21 +27,76 @@
 
 """
 
-import logging
+import ConfigParser
+import lsst.qserv.css
+from lsst.qserv.css import cssLib
 import os
 import time
 import unittest
-from kvInterface import KvInterface, KvException
+
+sqlConn = None
+testDatabaseName = "testCSSZ012sdrt"
+
+def getConfig():
+    """
+    @brief loads the configuration information needed to test KvInterface with a running mysql server
+    @return a formatted MySqlConfig object
+    """
+    cfgFile = os.path.join(os.path.expanduser("~"), ".lsst", "KvInterfaceImplMySql-testRemote.txt")
+    cfgParser = ConfigParser.ConfigParser()
+    filesRead = cfgParser.read(cfgFile)
+    if not cfgFile in filesRead:
+        return None
+    cfg = cssLib.MySqlConfig()
+    cfg.username = cfgParser.get('mysql', 'user')
+    cfg.password = cfgParser.get('mysql', 'passwd')
+    cfg.hostname = cfgParser.get('mysql', 'host')
+    cfg.port = cfgParser.getint('mysql', 'port')
+    cfg.dbName = testDatabaseName
+    return cfg
+
+    
+def initConnection():
+    """
+    @brief connects to the database and sets up a table with the schema to run the tests
+    @return True if conected & schema loaded, else false
+    """
+    # need config without database name
+    sqlConfigLocal = sqlConfig
+    sqlConfigLocal.dbName = ""
+    sqlConn = cssLib.SqlConnection(sqlConfigLocal)
```

Comment:
is this supposed to be `global sqlConn`? OTOH `initConnection()` is not used anywhere, just drop it.